playkit/src/main/java/com/kaltura/playkit/player/PlayerController.java

```diff
@@ -777,6 +777,10 @@ public void setEventListener(PKEvent.RawListener eventListener) {
                         event = new PlayerEvent.VolumeChanged(player.getVolume());
                         break;
                     case PLAYBACK_INFO_UPDATED:
+                        if (player.getPlaybackInfo() == null) {
+                            log.e("PLAYBACK_INFO_UPDATED event received, but can not send event.");
```

Comment:
lets make it something like                            
 log.e("PLAYBACK_INFO_UPDATED event payload == null, ignoring event");